Bust Config.rb

```diff
@@ -157,8 +158,49 @@
     face_indexes: [3],
   },
 ]
+
+# map from clothed Simon to corresponding topless Simon
+Busty::TOPLESS_SIMON_MAP = {
+  "1 Simon dark eyes" => "1 Simon topless dark eyes",
+  "1 Simon dark eyes2" => "1 Simon topless dark eyes2",
+  "1 Simon dark" => "1 Simon topless",
+  "1 Simon dark2" => "1 Simon topless2",
+  "face002b" => "face002b topless",
+  "face002b2" => "face002b topless2",
+  "face002b dark" => "face002b topless dark eyes",
+  "face002b dark2" => "face002b topless dark eyes2",
+}
+Busty::TOPLESS_SIMON_MAP.each do |key, value|
+  Busty::FACE_TO_BUST[value] = "Simon topless"
+end
+# Topless young Simon needs a special beardless bust and shouldn't be autoswapped
+Busty::TOPLESS_SIMON_YOUNG = ["face002b topless","face002b topless dark eyes"]
+Busty::TOPLESS_SIMON_YOUNG.each do |face|
+  Busty::SUBSET_TO_BUST.insert(-1, { character_name: "Simon topless young", face_name: face, face_indexes: [7] })
+end
+
 # Special cheats for some characters
 class Window_Message < Window_Base
+  def face_to_display
+    topless_simon_face = Busty::TOPLESS_SIMON_MAP[$game_message.face_name]
+    if topless_simon_face
+      if $game_actors[2].character_index == 2 && 
+          ($game_message.face_index != 7 || !Busty::TOPLESS_SIMON_YOUNG.include?(topless_simon_face))
+        # Simon is using his almost naked sprite, override with topless faceset
+        return topless_simon_face
+      end
+    elsif $game_message.face_name == 'Uyae emo'
+      if $game_switches[2081] # HOME BASE
+        # reinstate Uyae's boob window after travelling the world between ch3-4
+        return 'Uyae3 emo'
+      elsif $game_switches[1481] # YHILIN III
+        # fix up Uyae's clothes after returning from first Zirantia trip
+        return 'Uyae2 emo'
+      end
+    end
+    return $game_message.face_name
+  end
+  
   def custom_bust_display_options
```

Comment:
`custom_bust_display_options` was a mistake and can now be removed (Vera is also a faceswap). Allows to get rid of `bust_display_options` while we're at it.